po/git-novice.ja.po

```diff
@@ -7676,15 +7665,15 @@ msgid ""
 "different aspects of Open Science in [this book](https://link.springer.com/"
 "book/10.1007/978-3-319-00026-8)."
 msgstr ""
-"This open model accelerates discovery:\n"
-"the more open work is,\n"
-"[the more widely it is cited and re-used](https://doi.org/10.1371/journal."
-"pone.0000308).\n"
-"However,\n"
-"people who want to work this way need to make some decisions\n"
-"about what exactly \"open\" means and how to do it. You can find more on the "
-"different aspects of Open Science in [this book](https://link.springer.com/"
-"book/10.1007/978-3-319-00026-8)."
+"このオープンモデルは発見を加速します:\n"
+"オープンな作品が多いほど、\n"
```

Comment:
"more" does not refer to number of research products (work), but how open they are.
```suggestion
"研究結果がオープンになっていればなっているほど、\n"
```